recipes/orekit/meta.yaml

```diff
@@ -0,0 +1,75 @@
+{% set name = "orekit" %}
+{% set version = "9.0" %}
+{% set sha256 = "d58439c548433adcda98e695be53e526ba940a4b9c44fb9a05d92cd495cdd47f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://www.orekit.org/git/orekit-labs/python-wrapper-main.git
+  git_tag: orekit-v9
+  sha256: {{ sha256 }}
+
+build:
+  skip: true  # [win32 or linux32]
+
+  number: 0
+
+  rpaths:
+    - lib/
+    - jre/lib/amd64/
+    - jre/lib/amd64/server/
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - toolchain
+    - jcc >=3
+    # Force anaconda openjdk
+    - openjdk 8.0.121 1
+
+  run:
+    - python
+    - jcc >=3
+    # Force anaconda openjdk
+    - openjdk 8.0.121 1
```

Comment:
Would raise an issue in the feedstock about this after it is created.

Reply:
Is the issue about the pinning of version? This should be changed back when the conda-forge version of openjdk is working. Alternatlively when there is an option to force using anaconda repo, there isn't today is it?